site.local.yml

```diff
@@ -0,0 +1,70 @@
+#
+# This is an example Antora playbook configuration file. By default it's not
+# the one that's recommended in the documentation. It's included as a sample or
+# demonstration of what is possible, along with links to the relevant sections
+# of the documentation.
+#
+
+#
+# In this section, you can configure the core playbook configuration settings.
+# This section covers the site's title, base URL, and site start page. For more
+# information, see https://docs.antora.org/antora/1.0/playbook/configure-site/
+#
+site:
+  title: ownCloud Documentation
+
+content:
+  sources:
+  # This configuration enables what's referred to as "Author Mode". In this
+  # "mode", the local working copy is used to build the documentation. It
+  # doesn't refer to a specific branch, whether local or remote. This avoids
+  # generating only static assets, but no documentation. 
+  # For more information, see https://docs.antora.org/antora/1.0/playbook/author-mode/. 
+  # Normally, you would add standard content sources here. For more information
+  # about them, see https://docs.antora.org/antora/1.0/playbook/configure-content-sources/.
+  - url: ./
+    branches: HEAD
+  # Other components (modules) that you wish to include as part of this playbook.
+  - url: https://github.com/owncloud/android.git
+    branches:
+    - master-antora
+    start_path: docs/
+  - url: https://github.com/owncloud/ios.git
+    branches:
+    - master-antora
+    start_path: docs/
+  - url: https://github.com/owncloud/client.git
+    branches:
+    - master-antora
+    start_path: docs/
+
+#
+# In this section, you can configure the UI/theme for the documentation. This
+# covers the UI theme bundle to use, and whether to use a default page layout.
+# For more information, see
+# https://docs.antora.org/antora/1.0/playbook/configure-ui/
+#
+ui:
+  bundle:
+    url: https://github.com/owncloud/docs-ui/releases/download/1.0.0/ui-bundle.zip
+  output_dir: assets
+
```

Comment:
If that gets included, I question myself, and that may be out of that PR (or not) which components (directory or files) are mandatory needed in the docs repro and which once are or can be included by the .yml file.
I am thinking of the docs/antora-ui-default/ directory. Tese things seem to be double present?

Reply:
I don't follow, sorry.

Reply:
Note: the directory ``docs/antora-ui-default/`` has been removed in the meanwhile by PR https://github.com/owncloud/docs/pull/209

Reply:
I'm planning to remove bundle.url for the production playbook file, as it should be specified on the command-line by the build tool as part of the build pipeline.